### ISSUE TITLE
Windows: unable to detect system fonts that already exist

### ISSUE THREAD

**Issue by Intelligent2013** [fontist/fontist#175]
Looks like fontist can't detect bold, italic, bold/italic system fonts.

OS: Windows7 64-bit.
In the folder C:\Windows\Fonts\ there are system fonts files:
- arial.ttf
- arialbd.ttf
- arialbi.ttf
- ariali.ttf

File `arial.install.yml`:
```
Arial:
  - Regular
  - Bold
  - Italic
  - Bold Italic
```

Command: `fontist manifest-install --confirm-license arial.install.yml`

Output:
```
Downloading font "arial_black" from https://gitlab.com/fontmirror/archive/-/raw/master/arialb32.exe
Downloads: 0MB/0MB (100.00%) (cache)
Installing font "arial_black".
Fonts installed at:
- C:/Users/User/.fontist/fonts/AriBlk.TTF
Downloading font "arial_black" from https://gitlab.com/fontmirror/archive/-/raw/master/arialb32.exe
Downloads: 0MB/0MB (100.00%) (cache)
Installing font "arial_black".
Fonts installed at:
- C:/Users/User/.fontist/fonts/AriBlk.TTF
Downloading font "arial_black" from https://gitlab.com/fontmirror/archive/-/raw/master/arialb32.exe
Downloads: 0MB/0MB (100.00%) (cache)
Installing font "arial_black".
Fonts installed at:
- C:/Users/User/.fontist/fonts/AriBlk.TTF
---
Arial:
  Regular:
  - C:/Windows/Fonts/arial.ttf
  - C:/Windows/Fonts/arialbd.ttf
  - C:/Windows/Fonts/arialbi.ttf
  - C:/Windows/Fonts/ariali.ttf
  - C:/Windows/Fonts/ARIALN.TTF
  - C:/Windows/Fonts/ARIALNB.TTF
  - C:/Windows/Fonts/ARIALNBI.TTF
  - C:/Windows/Fonts/ARIALNI.TTF
  Bold: []
  Italic: []
  Bold Italic: []
```

Questions:
1. Why Arial **Black** font installed?
2. arialbd.ttf - is bold, but placed in Regular array, arialbi.ttf - is italic, but placed in Regular array, etc.

**Comment by ronaldtse** [comment]
> 1. Why Arial Black font installed?

This is a case of wrong matching, a Fontist bug.

> 2. arialbd.ttf - is bold, but placed in Regular array, arialbi.ttf - is italic, but placed in Regular array, etc.

Fontist bug.

Ping @alexeymorozov

**Comment by alexeymorozov** [comment]
Seems it's time to deal with matching in system directories. I will need some time to research why exactly this happens.

**Comment by ronaldtse** [comment]
The font name mismatch issue moved to #177.

**Comment by alexeymorozov** [comment]
To keep in mind: we have usage of an undocumented feature in metanorma-cc. It requests fonts without any style. May be just need to document it and cover with specs.

https://github.com/metanorma/metanorma-cc/blob/v1.6.1/lib/metanorma/cc/fonts_manifest.yaml

**Comment by alexeymorozov** [comment]
@ronaldtse Do we need to support search by filename?

```ruby
Fontist::Font.find("DejaVuSerif.ttf")
```

instead of

```ruby
Fontist::Font.find("DejaVu Serif")
```

**Comment by ronaldtse** [comment]
@alexeymorozov yes that would be a good feature, but instead of "find" maybe "find_by_filename"?

**Comment by alexeymorozov** [comment]
Released detection fix in v1.7.1 🚀

Will continue work in #177
